docs/farming-&-staking/farming/advanced-cli/platforms/_linux.mdx

```diff
@@ -99,7 +99,7 @@ import styles from '@site/src/pages/index.module.css';
 ```
 
 :::note
-Using **run**, setting **--base-path** and specifying **--chain** became mandatory starting with Gemini 3h. 
+Using **run**, setting **--base-path** and specifying **--chain** became mandatory starting with Gemini 3h. The parameter `--validator` was switched to `--farmer` at the same time.
```

Comment:
Hm, we had a similar sentence in this note before. Sounds like a we might have a problem of not working of the most up to date master, it's the only reason this message could have disappeared.

Reply:
> Hm, we had a similar sentence in this note before. Sounds like a we might have a problem of not working of the most up to date master, it's the only reason this message could have disappeared.

This arrose from me running update-version command to bring pre-release and latest inline. Looks like it was still present on the other version. Fixed now!